feat: Add theme options for cheatsheet (#346)
ansys/ansys-sphinx-theme: doc/source/user_guide/options.rst

    If you do not set the "use_meilisearch" option,
    ``ansys-sphinx-theme`` uses the default search functionality 
    inherited from the PyData Sphinx Theme.

Cheatsheet
----------

The theme option cheatsheet provides a quick reference guide for users to easily access 
and understand the available options in the Sphinx theme.

To enable the cheatsheet, you can add a child 
dictionary called ``cheatsheet`` to the ``html_theme_options``
dictionary in the ``conf.py`` file. 

The dict should contain the following keys:

#. ``url``: The URL of the cheatsheet for downloading.
#. ``title``: Title of the cheatsheet.

> #. ``image``: The thumbnail image for the cheatsheet.

Reviewer: I think thumbnail describes better the purpose of this variable:

```suggestion
#. ``thumbnail``: The thumbnail image for the cheatsheet.
```

> #. ``local_download``: Specifies whether to download the cheatsheet locally during the build. 

Reviewer: Can I suggest to use a name more compliant with boolean variables? Something like:

```suggestion
#. ``needs_download``: Specifies whether to download the cheatsheet locally during the build. 
```

> Cheatsheet
> ----------
> 
> The theme option cheatsheet provides a quick reference guide for users to easily access 
> and understand the available options in the Sphinx theme.
> 
> To enable the cheatsheet, you can add a child 
> dictionary called ``cheatsheet`` to the ``html_theme_options``
> dictionary in the ``conf.py`` file. 
> 
> The dict should contain the following keys:
> 
> #. ``url``: The URL of the cheatsheet for downloading.
> #. ``title``: Title of the cheatsheet.
> #. ``thumbnail``: The thumbnail image for the cheatsheet.
> #. ``needs_download``: Specifies whether to download the cheatsheet locally during the build. 
>    If set to `True`, the cheatsheet is downloaded to `_build/html/_static/`, otherwise, 
>    it is accessed directly from the provided URL. ``needs_download`` defaults to `False`.
> #. ``pages``: A list of pages to include in the cheatsheet (optional). If not provided, 
>    the cheatsheet includes the index page of the documentation.

Reviewer 2: ```suggestion
Cheat sheets
------------

The ``cheatsheet`` HTML theme option can be used to enable a cheat sheet in the Ansys Sphinx theme.

To enable a cheatsheet, in the ``conf.py`` file for your PyAnsys
library, you can add a child dictionary named ``cheatsheet`` to
the ``html_theme_options`` dictionary. 

This dictionary should contain these keys, in the order given:

#. ``url``: URL of the cheat sheet for downloading.
#. ``title``: Title of the cheat sheet.
#. ``thumbnail``: Thumbnail image for the cheat sheet.
#. ``needs_download``: Whether to download the cheat sheet locally during the build. The default is ``False``, in which case the cheat sheet is accessed directly from the provided URL. If ``True``, the cheat sheet is downloaded to the ``_build/html/_static/`` directory. 
#. ``pages``: List of pages to include in the cheat sheet (optional). If no list is provided, the cheat sheet includes the index page of the documentation.
```
Cheat sheet (2 words) in descriptive text.
In second-to-last-bullet, does "build" refer to the documentation build? Maybe this should be more specific. In the last bullet, I'm not understanding what you mean. What list of pages can be included in the cheat sheet? How does the cheat sheet include the index page of the documentation. Do you mean "List of pages to include the cheat sheet on."?  I think that this is likely.

.. code-block:: python

    html_theme_options = (
        {
            "cheatsheet": {

>                 "url": "<your cheatsheet URL>",
>                 "title": "<title of your cheatsheet>",
>                 "thumbnail": "<image URL>",
>                 "needs_download": True,  # True if you want to download the cheatsheet locally in ``_build/html/_static/``
>                 "pages": "<list of pages to include in the cheatsheet>",  # Optional

Reviewer 2: ```suggestion
                "url": "<your cheat sheet URL>",
                "title": "<title of your cheat sheet>",
                "thumbnail": "<image URL>",
                "needs_download": True,  # True if you want to download the cheatsheet locally in the ``_build/html/_static/`` directory.
                "pages": "<list of pages to include the cheat sheet on>",  # Optional
```
